app/javascript/article-form/components/Options.jsx

```diff
@@ -55,59 +80,73 @@ export const Options = ({
     );
   }
   return (
-    <Dropdown
-      className={
-        moreConfigShowing &&
-        'inline-block bottom-2 s:bottom-100 left-2 s:left-0 right-2 s:left-auto'
-      }
-      style={{ zIndex: 100 }}
-    >
-      <h3 className="mb-6">Post options</h3>
-      <div className="crayons-field mb-6">
-        <label htmlFor="canonicalUrl" className="crayons-field__label">
-          Canonical URL
-        </label>
-        <p className="crayons-field__description">
-          Change meta tag
-          {` `}
-          <code>canonical_url</code>
-          {` `}
-          if this post was first published elsewhere (like your own blog).
-        </p>
-        <input
-          type="text"
-          value={canonicalUrl}
-          className="crayons-textfield"
-          placeholder="https://yoursite.com/post-title"
-          name="canonicalUrl"
-          onKeyUp={onConfigChange}
-          id="canonicalUrl"
-        />
-      </div>
-      <div className="crayons-field mb-6">
-        <label htmlFor="series" className="crayons-field__label">
-          Series
-        </label>
-        <p className="crayons-field__description">
-          Will this post be part of a series? Give the series a unique name.
-          (Series visible once it has multiple posts)
-        </p>
-        <input
-          type="text"
-          value={series}
-          className="crayons-textfield"
-          name="series"
-          onKeyUp={onConfigChange}
-          id="series"
-          placeholder="..."
-        />
-        {existingSeries}
-      </div>
-      {publishedField}
-      <Button className="w-100" data-content="exit" onClick={toggleMoreConfig}>
-        Done
-      </Button>
-    </Dropdown>
+    <div className="s:relative">
+      <Button
+        id="post-options-btn"
+        variant="ghost"
+        contentType="icon"
+        icon={Icon}
+        title="Post options"
+      />
+
+      <Dropdown
+        triggerButtonId="post-options-btn"
+        dropdownContentId="post-options-dropdown"
+        dropdownContentCloseButtonId="post-options-done-btn"
+        className="bottom-2 s:bottom-100 left-2 s:left-0 right-2 s:left-auto"
+        style={{ zIndex: 100 }}
```

Comment:
I tested this without the `zIndex` on Desktop and mobile and the `z-index` does not appear to be needed unless there is some edge case I'm not aware of. cc: @fdoxyz

Reply:
Hmmm good point. The crayons `Dropdown` has z-index 400 by default, and I'm not sure of a reason why we'd need to lower the value here